lib/bundle/brew_dumper.rb

```diff
@@ -75,6 +82,7 @@ def formula_inspector(f)
       {
         :name => f["name"],
         :full_name => f["full_name"],
+        :aliases => f["aliases"],
```

Comment:
I don't think we need to handle that case for now unless it's specifically requested.